java/test/jmri/jmrix/can/cbus/CbusThrottleManagerTest.java

```diff
@@ -184,8 +185,8 @@ public void testIncomingSpeedDirection() {
         
         Assert.assertEquals("speed setting",0.0f,cbtmb.getThrottleInfo(addr,"SpeedSetting"));
         Assert.assertEquals("speed increment",(1.0f/126.0f),cbtmb.getThrottleInfo(addr,"SpeedIncrement"));
-        Assert.assertEquals("speed step mode",CbusConstants.CBUS_SS_128,cbtmb.getThrottleInfo(addr,"SpeedStepMode"));
-        
+        Assert.assertEquals("speed step mode",SpeedStepMode.SpeedStepMode128,cbtmb.getThrottleInfo(addr,"SpeedStepMode"));
```

Comment:
This seems like it was only working before due to a bug. I cannot tell if the updated version is the original intended behavior or not.

Reply:
Hi @trainman419 ,

Expected behaviour is to create CBUS Throttles with 128 SS by default, after the Throttle has been successfully created the SS can then be changed.
Listening for CBUS speed-step changes from a Command Station or another Throttle is on my to-do list though I've yet to see a user request for this,
Steve.

Reply:
Ok, then it looks like this is working as intended. Thanks!